### PR TITLE
Clarify that undoing a shutdown might not be possible

### PR DESCRIPTION
**Pull request by turt2live**
### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)
* [x] Code style is correct (run the [linters](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#code-style))

### CODE REVIEW

File: docs/admin_api/shutdown_room.md [hunk 1]
```diff
@@ -79,13 +79,20 @@ Response:
 the structure can and does change without notice.
 
 First, it's important to understand that a room shutdown is very destructive. Undoing a shutdown is not as simple as pretending it
-never happened - work has to be done to move forward instead of resetting the past.
+never happened - work has to be done to move forward instead of resetting the past. In fact, in some cases it might not be possible
+to recover at all:
 
-1. For safety reasons, it is recommended to shut down Synapse prior to continuing.
+* If the room was invite only, your users will need to be re-invited.
```
Comment (richvdh) [inline docs/admin_api/shutdown_room.md:85]:
```suggestion
* If the room was invite-only, your users will need to be re-invited.
```

File: docs/admin_api/shutdown_room.md [hunk 2]
```diff
@@ -79,13 +79,20 @@ Response:
...
+* If the room became empty, it'll be impossible to rejoin.
```
Comment (richvdh) [inline docs/admin_api/shutdown_room.md:86]:
```suggestion
* If the room no longer has any members at all, it'll be impossible to rejoin.
```

File: docs/admin_api/shutdown_room.md [hunk 3]
```diff
@@ -79,13 +79,20 @@ Response:
...
+* If the room was purged from your server, it may be inaccessible.
```
Comment (richvdh) [inline docs/admin_api/shutdown_room.md:87]: I don't think this makes any difference?

File: docs/admin_api/shutdown_room.md [hunk 4]
```diff
@@ -79,13 +79,20 @@ Response:
...
+
```
Comment (richvdh) [inline docs/admin_api/shutdown_room.md:88]: also:
```suggestion
 * the first user to rejoin will have to do so via an alias on a different server

```

File: docs/admin_api/shutdown_room.md (hunk 3 above)
Comment (turt2live) [reply docs/admin_api/shutdown_room.md:87]: fair

File: docs/admin_api/shutdown_room.md [hunk 5]
```diff
@@ -79,13 +79,20 @@ Response:
...
 
 First, it's important to understand that a room shutdown is very destructive. Undoing a shutdown is not as simple as pretending it
-never happened - work has to be done to move forward instead of resetting the past.
+never happened - work has to be done to move forward instead of resetting the past. In fact, in some cases it might not be possible
+to recover at all:
 
-1. For safety reasons, it is recommended to shut down Synapse prior to continuing.
+* If the room was invite-only, your users will need to be re-invited.
+* If the room no longer has any members at all, it'll be impossible to rejoin.
+  * The first user to rejoin will have to do so via an alias on a different server.
```
Comment (richvdh) [inline docs/admin_api/shutdown_room.md:87]: is the change in list level deliberate?

Comment (turt2live) [reply docs/admin_api/shutdown_room.md:87]: nope, I ran an autoformatter after applying suggestions and it decided that 1 space meant 2. Will push it back

**Pull request merged**